src/reformat.c

```diff
@@ -763,7 +763,7 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image,
                     G = Y - ((2 * ((kr * (1 - kr) * Cr) + (kb * (1 - kb) * Cb))) / kg);
                 }
             } else {
-                // Monochrome: just populate all channels with luma (identity mode is irrelevant)
+                // Monochrome: just populate all channels with luma (state->mode is irrelevant)
```

Comment:
Alternatively we could say "reformat mode is irrelevant". I use "state->mode" because the `if (hasColor)` code above (lines 735-764) uses different formulas depending on `state->mode`.

Reply:
i think the comment you have now is fine.